app/services/media-gallery/media-gallery.ts

```diff
@@ -6,15 +6,15 @@ import { UserService } from 'services/user';
 import { HostsService } from 'services/hosts';
 import { WindowsService } from 'services/windows';
 import uuid from 'uuid';
-import { stockImages, stockSounds } from './stock-library';
 import { $t } from '../i18n';
 
 export interface IMediaGalleryFile {
   href: string;
-  fileName: string;
+  filename: string;
```

Comment:
Is this a breaking change to the API?  We aren't trying to coordinate deploys here are we?  Because that won't work well.

Reply:
we didn't use an api before stock files were hard coded

Reply:
Oh right of course.  Got it.